### PR TITLE
Proxx 1

### LINKED ISSUE #1270: content: Making PROXX work on feature phones - part 1
**Author**: @surma
**Reviewer**: @kaycebasques  

**Target publish date:** As soon as it's done.
- [ ] Check this box if this is a hard deadline.
- [x] Check this box if you'd like your reviewer to fix grammar and punctuation issues for you.

**Process**

- [x] 1. I've done my [proposal](https://docs.google.com/document/d/1wMGwJiijyYReMLup-HRoKDMMVSfqvAi-B9dW14o8BeQ/edit?usp=sharing)
- [x] 2. Proposal approved — _reviewer checks this_
- [x] 3. I've added a draft to my proposal doc
- [x] 4. Draft approved — _reviewer checks this_
- [x] 5. I've submitted a pull request #1531 
- [ ] 6. Pull request approved — _reviewer checks this_

### PR DESCRIPTION
**Pull request by surma**
Fixes #1270

### CODE REVIEW

**Comment by robdodson** [comment]
looks like there are some markdown issues that travis caught. Mind fixing those @surma ?

I'll see about assigning a reviewer when I get in tomorrow.

**Comment by surma** [comment]
Alright, fixed those!

Question, tho: Why don’t I get these errors locally? Shouldn’t `npm run build` or at least `npm test` fail with the same message locally if CI is gonna fail? 

Also, at that point, it might be nice to offer a `npm run format` or something that runs `remark` and auto-fixes the problems that are machine fixable. Should I open an issue for this?

**Comment by robdodson** [comment]
> Question, tho: Why don’t I get these errors locally? Shouldn’t npm run build or at least npm test fail with the same message locally if CI is gonna fail?

Just tested and `npm run build` did fail for me locally when I made a markdown mistake. I think it should also block pushing. Have you synced with master lately?

File: src/site/content/en/blog/proxx-1/index.md [hunk 1]
```diff
@@ -0,0 +1,553 @@
...
+  </picture>
+  <figcaption class="w-figcaption">
+    The landing page of PROXX. Only critical components are used here.
+  </figcaption>
+</figure>
+
+To understand where our bundle size is coming from we can use a [source map explorer] or a similar tool to break down what the bundle consists of. As predicted, our bundle contains the game logic, the rendering engine, the win screen, the lose screen and a bunch of utilities. Only a small subset of these modules are needed for the landing page. Moving everything that is not strictly required for interactivity into a lazily-loaded module will decrease TTI _significantly_.
+
+<figure class="w-figure w-figure--center">
+  <style>
```
Comment (robdodson) [inline src/site/content/en/blog/proxx-1/index.md]: I was wondering if it's possible to turn this into an image? My main concern is around inadvertently breaking the layout during some far future update to the site's design or CSS. In general, we've tried to discourage folks from doing inline code for this reason.

File: src/site/content/en/blog/proxx-1/sourcemap.html [hunk 2]
```diff
@@ -0,0 +1,210 @@
+<style>
```
Comment (robdodson) [inline src/site/content/en/blog/proxx-1/sourcemap.html]: Are you able to remove this file? It looks like it was inlined directly into the article instead.

File: src/site/content/en/blog/proxx-1/index.md (hunk 1 above)
Comment (surma) [reply src/site/content/en/blog/proxx-1/index.md]: Hm, I can turn it into a foreign object SVG. Let me try that.

File: src/site/content/en/blog/proxx-1/sourcemap.html (hunk 2 above)
Comment (surma) [reply src/site/content/en/blog/proxx-1/sourcemap.html]: Turned this file into an SVG to address the above issue.

File: src/site/content/en/blog/proxx-1/index.md (hunk 1 above)
Comment (surma) [reply src/site/content/en/blog/proxx-1/index.md]: Works well. Done.

**Comment by robdodson** [comment]
@kaycebasques wanted to bubble this to the top of your stack since you've been out. Do you think you might have time to take a look today or tomorrow?

**Comment by kaycebasques** [comment]
Yes I'll look at it now.

**Comment by surma** [comment]
@kaycebasques PTAL!

**Comment by robdodson** [comment]
Btw the url for the post will be `web.dev/proxx-1`. I was wondering if that's the url you wanted or if you'd want to rename it to something else?

**Comment by surma** [comment]
Good shout! Renamed the folder :)

**Pull request merged**